src/OrbitQt/orbitmainwindow.cpp

```diff
@@ -924,6 +928,8 @@ void OrbitMainWindow::on_actionCaptureOptions_triggered() {
 
   orbit_qt::CaptureOptionsDialog dialog{this};
   dialog.SetCollectThreadStates(settings.value(kCollectThreadStatesSettingKey, false).toBool());
+  dialog.SetEnableApi(settings.value(kEnableApiSettingKey, false).toBool());
```

Comment:
Do we want the default to be false? Wouldn't the user expect to get manual instrumentation by default if they introduced it in their code? Or is there a reason why it should be false by default?

Reply:
Yes you're right, I changed it to default to true, and same for introspection to maintain the same behavior as before by default. Thanks.